app/components/documents/batch-details/document-details-row.hbs

```diff
@@ -7,11 +7,12 @@
       <AuCheckbox
         @checked={{@isSelected}}
         @onChange={{fn @onSelect @row}}
+        @disabled={{not @row.canBeEditedOrDeleted}}
       />
     </td>
   {{/if}}
   <td>
-    {{#if @isEditingEnabled}}
+    {{#if @row.canBeEditedOrDeleted}}
```

Comment:
purely looking at the incoming parameters I would argue that this replaces a permission "manage-documents" with a check on data (which could mean non-editors can edit/delete documents now when there is no signFlow attached.

Luckily, the 3 places where we use this batch doc edit component are all covered by the permission.
The edit button does not exist for them (look for `this.openBatchDetails`)
But these rows were built with a non-editor viewer in mind I think. (not used as such yet)
So I would check both `@isEditingEnabled` and the extra boolean